viewer/core/static/css/dataset-control.styl

```diff
@@ -0,0 +1,3 @@
+.dataset-selection {
+  width: 150px;
```

Comment:
Remove this please I've fixed it differently in something that will follow this patch.